README.md

```diff
@@ -268,6 +268,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [kemal-session](https://github.com/kemalcr/kemal-session) - Session handler for Kemal
  * [mime-types.cr](https://github.com/jwaldrip/mime-types.cr) - A port of the Ruby MIME-types library
  * [multi-auth](https://github.com/msa7/multi_auth) - Standardized multi-provider OAuth2 authentication (inspired by omniauth)
+ * [praetorian](https://github.com/ilanusse/praetorian) - Minimalist authorization library for Crystal inspired by Pundit.
```

Comment:
Can we remove "for Crystal" as it is implied in this scope? And please remove a full stop (dot at the end) for consistency.